src/cli/push.go

```diff
@@ -14,14 +15,17 @@ const pushLongDescription = `Command "push"
 Push configurations to the Keboola Connection project.
 Project's state will be overwritten to match the local files.
 
+You can specify an optional ["change description"].
+It will be visible in the config's versions.
+
 You can use the "--dry-run" flag to see
 what needs to be done without modifying the project's state.
 `
 
 func pushCommand(root *rootCommand) *cobra.Command {
 	force := false
 	cmd := &cobra.Command{
-		Use:   "push",
+		Use:   `push ["change description"]`,
```

Comment:
Prisla poziadavka (vid Jira) aby uzivatel mohol zadat svoju `changeDescription`, to je textova polozka v API, ... ktora sa nasledne zobrazi v UI, pri jednotlivych verziach config/row.